bullets/enemy.py

```diff
@@ -2,4 +2,31 @@
 
 
 class Enemy(Actor):
-    pass
+    def __init__(self, maxhealth, pcb1, pcb2, pcb3):
```

Comment:
To inherit all the `Actor` stuff (which admittedly isn't much rn), we need to call the `super()`'s `__init__` method.  You can see this [here in `Player`](https://github.com/TriCot-Game-Studio/bullets/blob/main/bullets/player.py#L5-L6).

Reply:
This would also require for `Enemy`'s `__init__()` to get a few more params.